platform/vendor/tekton/catalog/main/task/trivy-scanner/0.1/trivy-scanner.yaml

```diff
@@ -32,6 +32,14 @@ spec:
       description: Image or Path to be scanned by trivy.
       type: string
   steps:
+    - name: get-latest-cve-findings
+      image: $(params.TRIVY_IMAGE)
+      script: |
+        TRIVY_TEMP_DIR=$(mktemp -d)
```

Comment:
This won't work because it's a change against the vendored task. This would work if it was a patch similar to what's in: https://github.com/buildsec/frsca/tree/main/platform/components/tekton/tasks

This should change soon to something a bit nicer. Let me double check on the right approach here.

Reply:
Thanks so much; I've updated the patch according to your feedbacks from both of you, @mlieberman85 and @pxp928. And also, this is an excellent opportunity to add support for air-gap to the version of the trivy-scanner stored in Tekton Hub.